src/requirements.txt

```diff
@@ -1,15 +1,38 @@
+###ONLY KEEP WHAT IS IN THIS COMMENT SO CHANGES ARE CHECKED-IN
+###Flask==1.1.2
+###
+#### Used by reload_from_neo4j
+###py2neo==2021.0.1
+###
+#### Higher version (even 8.0.23) would cause issue because
+#### we still use RDS with older 5.6.mysql_aurora
+###mysql-connector-python==8.0.16
+###
+#### The commons package requires requests>=2.22.0
+###requests==2.25.1
+###
+#### The branch name of commons to be used during image build
+#### Default is master branch specified in docker-compose.yml if not set
+###git+https://github.com/hubmapconsortium/commons.git@${COMMONS_BRANCH}#egg=hubmap-commons
+
 Flask==1.1.2
 
 # Used by reload_from_neo4j
 py2neo==2021.0.1
 
-# Higher version (even 8.0.23) would cause issue because 
-# we still use RDS with older 5.6.mysql_aurora
 mysql-connector-python==8.0.16
 
 # The commons package requires requests>=2.22.0
 requests==2.25.1
 
 # The branch name of commons to be used during image build
 # Default is master branch specified in docker-compose.yml if not set
-git+https://github.com/hubmapconsortium/commons.git@${COMMONS_BRANCH}#egg=hubmap-commons
+###git+https://github.com/hubmapconsortium/commons.git@${COMMONS_BRANCH}#egg=hubmap-commons
```

Comment:
@kburke @yuanzhou I agree with Zhou- we should use a newer version of the mysql-connector, whatever you used, Karl on your local testing or higher if you think appropriate.
Also, I don't see why we can't pull hubmap-commons from PyPi?
Also, also, thinking we should test the service without including py2neo if that is only used by a script not included in the WS.  We could include a separate requirements.reload.from.neo.txt if needed, but I'm thinking that script can be removed because it is never used and certainly broken at this point.